kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -113,7 +113,11 @@
   export default {
     name: 'MediaPlayerIndex',
     components: { MediaPlayerFullscreen, MediaPlayerTranscript },
-    mixins: [commonCoreStrings, responsiveWindowMixin, responsiveElementMixin],
+    mixins: [commonCoreStrings, responsiveElementMixin],
+    setup() {
+      const { windowIsSmall } = useKResponsiveWindow();
```

Comment:
The `windowIsPortrait` property is also used, so must be destructured here and returned from the setup method.

See here: https://github.com/learningequality/kolibri/blob/release-v0.16.x/kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue#L207